### PR TITLE
Update and enhancement the command line 

### PR DESCRIPTION
**Pull request by michaelnabil230**
This PR is for removing the `spatie/php-structure-discoverer` and using the `Finder` from `Symfony`

### CODE REVIEW

**Comment by stevebauman** [comment]
Thanks for the PR! Tests are failing 👍

**Comment by michaelnabil230** [comment]
Hi @stevebauman, I fixed the issue and you can run the workflow again

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ composer require --dev stevebauman/autodoc-facades
 Inside the terminal:
 
 ```bash
-php artisan autodoc:facades {paths} {--only=} {--except=}
+php artisan autodoc:facades {path} {--only=} {--except=}
```
Comment (stevebauman) [inline README.md]: This is a breaking change, arg needs to remain `paths`.

File: README.md [hunk 2]
```diff
@@ -158,18 +158,18 @@ class ServiceManager
 
 To generate doc annotations for all facades in your `app` directory, supply "app" as the path:
 
-> All paths you provide that do not start with a directory separator will use the commands current working directory as the base path.
+> All path you provide that do not start with a directory separator will use the commands current working directory as the base path.
```
Comment (stevebauman) [inline README.md]: Please revert these `path` find and replace changes 👍

File: src/Commands/DocumentFacades.php [hunk 3]
```diff
@@ -30,51 +33,52 @@ class DocumentFacades extends Command
...
-        return collect(
-            Discover::in(...$this->paths())
-                ->classes()
-                ->extending(Facade::class)
-                ->get()
-        );
+        $except = $this->option('except');
+        $only = $this->option('only');
+
+        return collect(Finder::create()->files()->depth(0)->in($this->argument('path'))->name('*.php'))
```
Comment (stevebauman) [inline src/Commands/DocumentFacades.php]: This needs to support searching multiple paths, like was previously supported with Spatie Structure Discovery 👍

**Review by stevebauman — CHANGES_REQUESTED** [review]

**Review by stevebauman — APPROVED** [review]

**Comment by stevebauman** [comment]
Thanks @michaelnabil230! Really appreciate your contribution! 🙏

**Pull request merged**